Description: Add cfg for test requiring zdict_builder feature
Last-Update: 2022-09-02
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/tests.rs
+++ b/src/tests.rs
@@ -78,2 +78,3 @@
 #[test]
+#[cfg(feature = "zdict_builder")]
 fn test_dictionary() {
